paas/pay-apps.yml

```diff
@@ -322,6 +323,17 @@ applications:
       AWS_XRAY_CONTEXT_MISSING: LOG_ERROR
     routes:
     - route: pay-selfservice-((space)).london.cloudapps.digital
+  - <<: *applicationTemplate
+    name: notifications
+    docker:
+      <<: *dockerImage
+      image: govukpaypaas/notifications:latest-master
+    env:
+      CONNECTOR_URL: http://card-connector-((space)).apps.internal:8080
+      DD_CONNECTOR_URL: http://directdebit-connector-((space)).apps.internal:8080
+      TINI_SUBREAPER: ''
```

Comment:
what does this environment variable do?

Reply:
When you run a Docker container in PaaS it's actually being run by a tool called Garden which takes over as the PID1 process inside the container. Any zombie processes (subprocesses where the parent process has ended) in the container get reparented to the PID1 process. 

Setting `TINI_SUBREAPER` means it's able to have zombie processes reparented to it even if it's not PID1. I'm assuming it only gets zombie processes that are themselves a subsubsubprocess of `tini`.